### PR TITLE
Feat: connect ledger popups

### LINKED ISSUE #922: ️🛠 - Update ledger connection popup
### Task description

Use the new designs in the ledger connection popups

### Requirements

TBD

### LINKED ISSUE #999: Rebrand ConnectLedger Popop

### PR DESCRIPTION
**Pull request by Tuditi**
## Summary

closes: https://github.com/bloomwalletio/bloom/issues/922
closes: https://github.com/bloomwalletio/bloom/issues/999
...

### Platforms

> Please select any platforms where your changes have been tested.

-   **Desktop**
    -   [ ] MacOS
    -   [ ] Linux
    -   [ ] Windows

### Instructions

> Please describe the specific instructions, configurations, and/or test cases necessary to **test and verify** that your changes work as intended.

...

## Checklist

> Please tick the following boxes that are relevant to your changes.

-   [ ] I have followed the contribution guidelines for this project
-   [ ] I have performed a self-review of my own code
-   [ ] I have commented my code, particularly in hard-to-understand areas
-   [ ] I have added or modified tests that prove my changes work as intended
-   [ ] I have verified that new and existing unit tests pass locally with my changes
-   [ ] I have verified that my latest changes pass CI workflows for testing and linting
-   [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/desktop/components/popup/popups/ConnectLedgerPopup.svelte
Comment (MarkNerdi) [inline packages/desktop/components/popup/popups/ConnectLedgerPopup.svelte:1]: Please use the new `PopupTemplate` for this component :pray:

File: packages/desktop/views/dashboard/drawers/network-config/views/ConfirmLedgerEvmAddressDrawer.svelte [hunk 1]
```diff
@@ -17,34 +16,31 @@
...
-                            <Text fontWeight={FontWeight.medium} fontSize="13" color="gray-600">
-                                {localize('general.evmAddress')}
-                            </Text>
-                            <Text type={TextType.pre} fontWeight={FontWeight.medium} fontSize="15" classes="break-words"
-                                >{address}
-                            </Text>
-                        </div>
-                    </CopyableBox>
-                </Pane>
+                <Table orientation="vertical" {items} />
```
Comment (MarkNerdi) [inline packages/desktop/views/dashboard/drawers/network-config/views/ConfirmLedgerEvmAddressDrawer.svelte]:
```suggestion
                <Table orientation="vertical" items={
                [
        {
            key: localize('general.evmAddress'),
            value: address,
            copyable: true,
        },
    ]
                 } />
```

File: packages/desktop/views/dashboard/drawers/network-config/views/ConfirmLedgerEvmAddressDrawer.svelte [hunk 2]
```diff
@@ -17,34 +16,31 @@
...
     }
+
+    let items: IItem[]
+    $: items = [
+        {
+            key: localize('general.evmAddress'),
+            value: address,
+            copyable: true,
+        },
+    ]
```
Comment (MarkNerdi) [inline packages/desktop/views/dashboard/drawers/network-config/views/ConfirmLedgerEvmAddressDrawer.svelte]: move this into the html part

**Review by MarkNerdi — DISMISSED** [review]

**Review by nicole-obrien — CHANGES_REQUESTED** [review]
I think the ledger SVGs should me transparent and only include the ledger device.
The text should be a pill component 🙏

**Comment by nicole-obrien** [comment]
Unable to build

**Review by nicole-obrien — APPROVED** [review]

**Pull request merged**